.circleci/config.yml

```diff
@@ -348,13 +369,13 @@ workflows:
             - run: 'rm -rf ~/.yarn && npm i -g yarn && yarn -v && yarn global
                 add wait-on' # Use yarn latest
           yarn: true
-          record: false
+          record: true
           store_artifacts: false
           working_directory: platform/viewer
-          build: npx cross-env QUICK_BUILD=true yarn run build:package
+          build: npx cross-env QUICK_BUILD=true APP_CONFIG=config/dicomweb-server.js yarn run build:package
           start: yarn run test:e2e:serve
           spec: 'cypress/integration/common/**/*,cypress/integration/script-tag/**/*'
-          wait-on: 'http://localhost:3000'
+          wait-on: 'http://localhost:3000/index.umd.js'
```

Comment:
I'm not sure why this change is required.

Reply:
it's not, I was debugging. You don't need to review anything at this time it's all garbage while I was trying to debug the test running issues, I'll squash it and remove unnecessary stuff.